e/esc/esc_8.5.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+# Package	: esc
+# Version	: v0.2.0
+# Source repo	: https://github.com/mjibson/esc.git
+# Tested on	: ubi 8.5
+# Language      : go
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: saraswati patra <saraswati.patra@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
```

Comment:
we already have this on line #1, not needed again

Reply:
> we already have this on line #1, not needed again

Thanks priya ,it was typo.done now looking f9.